src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -16,8 +16,12 @@ description: Summary of technical specifications of Stacks 2.0
   - 500 STX/block for following 4 years
   - 250 STX/block for subsequent 4 years
   - 125 STX/block in perpetuity after that
+- Coinbase rewards accumulate for "missed sortitions": If a Bitcoin block has no sortition (at height N), then any Stacks block mined in a subsequent sortition that builds off of any Stacks chain tip that existed at the penultimate sortition (at height N-1) may claim its coinbase. This encourages miners to keep mining even if Bitcoin fees are high.
+- Initial mining bonus: This is a special case of the above to incentivize early miners. Coinbase for all burnchain blocks between the first burn block height (to be chosen by independent miners as part of the Stacks 2.0 launch) and the first sortition winner accumulate and are distributed to miners over a fixed window (to be determined). For instance, say burn block height is 10000 and first sortition is at block 10500 and distribution window is 100 blocks, then coinbase for the first 500 blocks (10500 - 10000) will be distributed evenly to miners who win sortition over the subsequent 100 blocks.
```

Comment:
`10000` > `10,000` (in several places in the para)

Reply:
Sorry, are you saying we prefer "10000" over "10,000" or the other way around?